lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -998,12 +999,12 @@ XcodeSDK SymbolFileDWARF::ParseXcodeSDK(CompileUnit &comp_unit) {
   const char *sdk = cu_die.GetAttributeValueAsString(DW_AT_APPLE_sdk, nullptr);
   if (!sdk)
     return {};
-  std::string sysroot =
+  llvm::StringRef sysroot =
```

Comment:
Now that this is a `StringRef` we don't need the move down on line 1021. Though I'd double-check that the lifetimes

Reply:
thanks for pointing out the `move`.

> double-check that the lifetimes make sense

It appears fine, and the tests pass. Is that enough or should we test this more in some way?